src/main/java/net/geforcemods/securitycraft/util/Reinforced.java

```diff
@@ -21,6 +21,11 @@
 	 */
 	boolean hasTint() default true;
 
+	/**
+	 * @return true if the BlockItem of this block should be automatically registered, false if not
+	 */
+	boolean registerBlockItem() default true;

```

Comment:
Move this to the bottom, please. Logically, `hasTint` and `tint` belong together.